drivers/at86rf2xx/at86rf2xx.c

```diff
@@ -106,6 +106,18 @@ void at86rf2xx_reset(at86rf2xx_t *dev)
     tmp &= ~(AT86RF2XX_TRX_CTRL_1_MASK__IRQ_MASK_MODE);
     at86rf2xx_reg_write(dev, AT86RF2XX_REG__TRX_CTRL_1, tmp);
 
+    /* configure smart idle listening feature */
+#if AT86RF2XX_SMART_IDLE_LISTENING
```

Comment:
@Hyungsin @PeterKietzmann hmmm should this be compile time configuration? are there any use cases where it could be desired to turn it off from the code?